apps/site/public/glimm.js

```
@@ -0,0 +1,1829 @@
+  const compile = (type, src) => {
+    const s = gl.createShader(type);
+    gl.shaderSource(s, src);
+    gl.compileShader(s);
+    if (!gl.getShaderParameter(s, gl.COMPILE_STATUS)) {
+      console.error("[glimm] shader compile error:", gl.getShaderInfoLog(s));
+    }
+    return s;
+  };
+  const vs = compile(gl.VERTEX_SHADER, VS);
+  const fs = compile(gl.FRAGMENT_SHADER, FS);
+  const prog = gl.createProgram();
```

ESLint

[error] Insert `··`

(prettier/prettier)